exaparser/cli.py

```diff
@@ -6,22 +6,39 @@
 
 
 @click.group()
+@click.pass_context
 @click.option('-c', '--config', help='full path to config file', type=click.Path(exists=True))
-def main(config):
+def main(ctx, config):
     if config:
         ExaParserConfig.read(config)
+    ctx.ensure_object(dict)
+    ctx.obj['handlers'] = ExaParserConfig.get("global", "data_handlers").replace(" ", "").split(",")
 
 
 @main.command()
+@click.pass_context
 @click.argument('name')
 @click.argument('work-dir', type=click.Path(exists=True, file_okay=False, resolve_path=True))
-def job(name, work_dir):
+def job(ctx, name, work_dir):
     """Parse a job.
 
     Parses the output of job with NAME in directory WORK_DIR. For example:
 
         ./bin/exaparser job-123 /path/to/workdir/
     """
     job = Job(name, work_dir)
-    for handler in ExaParserConfig.get("global", "data_handlers").replace(" ", "").split(","):
+    for handler in ctx.obj['handlers']:
         get_data_handler(handler, job).handle()
+
+
+@main.command()
+@click.pass_context
+@click.argument('work-dir', type=click.Path(exists=True, file_okay=False, resolve_path=True))
+def structures(ctx, work_dir):
+    """Parse a working directory for structure(s) data.
```

Comment:
(1) People do both, but we used to add a newline after `"""` before, looks a bit cleaner. https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
(2) Args with types?

Reply:
(1) Might be worthwhile to install a linter or even auto-style tool so that this is automatically adjusted pre- or post-commit?
(2) The types are documented as part of the click decorators.